examples/datetime-timezone/pyproject.toml

```
@@ -0,0 +1,12 @@
+[project]
+name = "example-datetime-timezone"
+version = "0.1.0"
+description = "Datetime Timezone Example."
+readme = "README.md"
+requires-python = ">=3.9"
+dependencies = [
+    "flask-admin[sqlalchemy]",
```

This example doesn't run yet:

```
flask-admin/examples/datetime-timezone on  chore/uv-migration/examples is 📦 v0.1.0 via 🐍 v3.13.4
❯ uv run main.py
Using CPython 3.9.6 interpreter at: /Library/Developer/CommandLineTools/usr/bin/python3
Creating virtual environment at: .venv
Installed 14 packages in 8ms
Traceback (most recent call last):
  File "/Users/sam/git/flask-admin/examples/datetime-timezone/main.py", line 9, in <module>
    from flask_admin.contrib.sqla import ModelView
  File "/Users/sam/git/flask-admin/flask_admin/contrib/sqla/__init__.py", line 2, in <module>
    from .view import ModelView
  File "/Users/sam/git/flask-admin/flask_admin/contrib/sqla/view.py", line 33, in <module>
    from flask_admin.contrib.sqla import form
  File "/Users/sam/git/flask-admin/flask_admin/contrib/sqla/form.py", line 10, in <module>
    from sqlalchemy_utils import ChoiceType
ModuleNotFoundError: No module named 'sqlalchemy_utils'
```

```suggestion
    "flask-admin[sqlalchemy-with-utils]",
```